akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -17,24 +17,24 @@ import scala.collection.immutable
 import scala.concurrent.duration.FiniteDuration
 import scala.util.control.NonFatal
 
-/**
- * INTERNAL API
- */
-private[persistence] object Eventsourced {
+/** INTERNAL API */
+@InternalApi
+private[akka] object Eventsourced {
```

Comment:
Was it necessary to open this up?

Reply:
it was at some point, not anymore! thanks, will revert thi change :)